web/config.js

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Can we generate a Go file in the future so we don't have to embed the endpoints in the command line tool?

Reply:
Yeah absolutely!